nexus/tests/integration_tests/ip_pools.rs

```diff
@@ -275,69 +282,226 @@ async fn test_ip_pool_basic_crud(cptestctx: &ControlPlaneTestContext) {
         .expect("Expected to be able to delete an empty IP Pool");
 }
 
+/// The internal IP pool, defined by its association with the internal silo,
+/// cannot be interacted with through the operator API. CRUD operations should
+/// all 404 except fetch by name or ID.
+#[nexus_test]
+async fn test_ip_pool_service_no_cud(cptestctx: &ControlPlaneTestContext) {
```

Comment:
Is `cud` here short for create, update, delete?

Reply:
Yes. This is the sort of shenanigans that happen when you are not around